### ISSUE TITLE
Can anyone confirm that important/strategy doesn't work?

### ISSUE THREAD

**Issue by merijnponzo** [mhsdesign/jit-browser-tailwindcss#13]
Hello,

I try to add the [selectorstrategy](https://tailwindcss.com/docs/configuration#selector-strategy) to prevent style conflicts in my app. 

but it doesn't seem to add a class before the tailwind classes

```

async function createTailwind(content, prefix) {
  // get config from theme.json
  const config = configParser.tailwindTheme(themeJson, false);
  const configResult = createConfig(config, prefix);

  console.log(configResult)

  const tailwind = await window.createTailwindcss({
    tailwindConfig: configResult,
  });

  const css = await tailwind.generateStylesFromContent(
    `
    @tailwind components;
    @tailwind utilities;
    `,
    [content]
  );

  return css;
}
```

So my config object from createConfig
<img width="533" alt="Screenshot 2023-07-11 at 09 47 06" src="https://github.com/mhsdesign/jit-browser-tailwindcss/assets/23075300/20064ef2-46d1-4f0f-b1af-63aec62aa2ec">

however, no strategy prefix is added:
<img width="821" alt="Screenshot 2023-07-11 at 09 48 30" src="https://github.com/mhsdesign/jit-browser-tailwindcss/assets/23075300/d5206e92-2952-44a1-9c33-ca5351773c83">

**Comment by merijnponzo** [comment]
i realise i didn't add #style823 to my div yet

**Comment by mhsdesign** [comment]
so is it solved then? ^^ Fyi were using tailwind v3.3.0

**Comment by merijnponzo** [comment]
Yes it's solved and adding strategy prefixes to the classnames. Have a great day
